### ISSUE TITLE
Search engine only works when installed as root (Linux w/ guix install)

### ISSUE THREAD

**Issue by joseph8th** [atlas-engineer/next#206]
Love this project. Noticed that the search engine redirect (i.e., `C-l` > "s search something") only seemed to work when I did `sudo guix install sbcl-next`. If I installed as user, it would not redirect the search. If I did `M-l` first, I just got `about:blank`. With `C-l` it just concats my search string, including the initial "s", to the end of the current URL. 

I ran it in the terminal, and got the following on attempting search engine search:
```
 <INFO> [22:47:03] next remote.lisp (request.resource) -
  Forwarding s guix back to platform port
```

I tried setting my search engines per the related issue (adding search engines in `init.lisp`) but no dice. I might try installing from source (interested in the code anyway) but thought it was worth mentioning here.

**Comment by Ambrevar** [comment]
I think I understand the problem:  "s" is no longer a prefix for the
default search engine.
Just type what you want and press C-RET to search with the default search
engine.

Notice the difference:

- RET will open the completion candidate.
- C-RET will open whatever input you've typed.

The reason why you had a different experience as root is because there
was no history database and RET would fall back on the input since there
was no candidates.  Anyways, I strongly discourage you from running Next
as root! ;)

Special bonus: there is a "wiki" search engine to search on Wikipedia.

I reckon this is a bit confusing for newcomers.  I'm thinking we should
do like Helm and have the raw input as part of the candidates at the
top.

Thoughts? @jmercouris ?

**Comment by jmercouris** [comment]
hi @ambrevar, can you expand what you mean by having the “raw input” as part of the candidates at the top? a screenshot perhaps?

thanks,

-John

**Comment by Ambrevar** [comment]
Recipe:

- C-l foo

Minibuffer result I'm suggesting:

```
Open URL in buffer: foo
--------------------------------------------------------------------------------
foo
http://some.foo.domain.org/
http://some.other.domain.org/foo
...
```

**Comment by jmercouris** [comment]
how does it help to see the raw input as the first selection? are you saying so that a user can just press RET instead of having to press C-RET?

**Comment by Ambrevar** [comment]
Yes, and to make it obvious that they can send the raw input at all.
I don't expect many users to know about C-RET (took me an embarrassingly
long time to realize its existence :p)

**Comment by Ambrevar** [comment]
@jmercouris: Thoughts?

**Comment by jmercouris** [comment]
@Ambrevar I thought I had answered this, sorry about that! Sometimes I answer things in my head, but don't actually write them down.

I think that is a good idea, I quite like it! I can't think of any downsides to it (other than a little bit of screen real estate being consumed).

**Comment by Ambrevar** [comment]
Acknowledged!

**Comment by joseph8th** [comment]
Thanks!

**Comment by Ambrevar** [comment]
Fixed in a11666a0c28a41b8b529859423656a8068ebec94.
